i18n_subsites/localizing_using_jinja2.rst

```diff
@@ -6,11 +6,11 @@ Localizing themes with Jinja2
 ---------------------
 
 To enable the |ext| extension in your templates, you must add it to
-``JINJA_EXTENSIONS`` in your Pelican configuration
+``JINJA_ENVIRONNEMENT`` in your Pelican configuration
 
 .. code-block:: python
 
-  JINJA_EXTENSIONS = ['jinja2.ext.i18n', ...]
+	JINJA_ENVIRONMENT = {'extensions': ['jinja2.ext.i18n',]}
```

Comment:
It seems you increased the indent here unnecessarily. Please revert to previous indentation.